lib/puppet-lint/plugins/topscope_variable.rb

```diff
@@ -3,9 +3,9 @@ def check
     class_list = (class_indexes + defined_type_indexes)
     # do not check if the code is not part of a class
     return if class_list.first.nil?
```

Comment:
I wonder if this (and` class_list` in general) is still useful. If it is, this is probably a better way to write it (but not in this PR):
```suggestion
    return if class_list.empty?
```